tools/review/seed_feedback_from_github.py

```
@@ -30,17 +30,83 @@
     if token:
         req.add_header("Authorization", f"Bearer {token}")
     try:
         with urllib.request.urlopen(req) as r:
-            return json.loads(r.read().decode("utf-8"))
+            data = json.loads(r.read().decode("utf-8"))
+            headers = {k.lower(): v for k, v in r.headers.items()}
+            return data, headers
     except urllib.error.HTTPError as e:
         msg = e.read().decode("utf-8", errors="ignore")
-        raise SystemExit(f"GitHub API error {e.code} on {path}: {msg}")
+        raise SystemExit(f"GitHub API error {e.code} on {url}: {msg}")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)